Lib/profile/sample.py

```diff
@@ -53,6 +57,14 @@ def sample(self, collector, duration_sec=10):
                     break
                 except (RuntimeError, UnicodeDecodeError, MemoryError, OSError):
                     errors += 1
+                except ValueError as e:
+                    # Process is waiting to be reaped by the parent (us)
+                    if self._is_process_running():
+                        duration_sec = (
+                            current_time - start_time
+                        )
```

Comment:
Line length is fine here:
```suggestion
                    if self._is_process_running():
                        duration_sec = current_time - start_time
```